06_deploy_bootstrap_vm.sh

```diff
@@ -43,6 +43,79 @@ $GOPATH/src/github.com/openshift/installer/bin/openshift-install --dir ocp --log
 IGN_FILE="/var/lib/libvirt/images/${CLUSTER_NAME}-bootstrap.ign"
 sudo cp ocp/bootstrap.ign ${IGN_FILE}
 
+export DNS_VIP=$("${PWD}/nthhost" "$EXTERNAL_SUBNET" 2)
+# Generate bootstrap mDNS CoreDNS ignition
+KEEPALIVED_CONF_TEMPLATE="vrrp_instance API {
+    state BACKUP
+    interface \${INTERFACE}
+    virtual_router_id 51
+    priority 50
+    advert_int 1
+    authentication {
+        auth_type PASS
+        auth_pass cluster_uuid_api_vip
+    }
+    virtual_ipaddress {
+        \${MASTER_VIP}
```

Comment:
This also needs "scope link" added to it.

Reply:
Or maybe not. I was actually missing the scope on the API_VIP for the master nodes below too. Not sure whether it matters on the bootstrap.